tests/topotests/zebra_pref64/rx_ipv6_ra_8781.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
```

Comment:
This line should be called `#!/usr/bin/env python` to select the correct python
Test infrastructure (and potential users) may use a python venv with all the modules. However, the hard specification of the python location causes it to use the regular python outside the venv. 

This doesn't apply to normal `test_.....py` files as they are executed by the correct pytest or python binary, but this additional helper tool is invoked directly based on python in the shebang

Reply:
fixed